src/lib/draw/route/RouteControls.svelte

```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
   import { RouteTool } from "./route_tool";
   import KeyHandler from "../KeyHandler.svelte";
-  import CollapsibleCard from "../CollapsibleCard.svelte";
+  import CollapsibleCard from "../../CollapsibleCard.svelte";
```

Comment:
I moved this component up a level, since it's now used outside of draw modes. We could probably use with a little bit of organizing directory structure soon.